test/functional/cmdLineTests/gputests/playlist.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!--
+Copyright (c) 2022, 2022 IBM Corp. and others
+
+This program and the accompanying materials are made available under
+the terms of the Eclipse Public License 2.0 which accompanies this
+distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+or the Apache License, Version 2.0 which accompanies this distribution and
+is available at https://www.apache.org/licenses/LICENSE-2.0.
+
+This Source Code may also be made available under the following
+Secondary Licenses when the conditions for such availability set
+forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+General Public License, version 2 with the GNU Classpath
+Exception [1] and GNU General Public License, version 2 with the
+OpenJDK Assembly Exception [2].
+
+[1] https://www.gnu.org/software/classpath/license.html
+[2] http://openjdk.java.net/legal/assembly-exception.html
+
+SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+<playlist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../TKG/playlist.xsd">
+	<test>
+		<testCaseName>cmdLineTests_gputests</testCaseName>
+		<variations>
+			<variation>NoOptions</variation>
+		</variations>
+			<command>$(ADD_JVM_LIB_DIR_TO_LIBPATH) \
+				$(JAVA_COMMAND) \
+				$(JVM_OPTIONS) \
+				-DEXE=$(SQ)$(JAVA_COMMAND) $(JVM_OPTIONS)$(SQ) \
```

Comment:
Most of these parameters aren't used. I assume they were copied from some other test and they could be removed.

Reply:
I suggest it's easier to leave the unused definitions here than to remember to add them if `gputests.xml` is extended, but I could be convinced otherwise.

Reply:
They are quite specific to other testing. The only one I could see leaving is the FIBTARGET.

Reply:
Ok, I'll remove as many as I can.

Reply:
The only one that was needed was `EXE`: the others have been removed.